packages/core/src/Stores/portfolio-store.js

```diff
@@ -342,9 +343,11 @@ export default class PortfolioStore extends BaseStore {
                 sell_price: response.sell.sold_for,
                 transaction_id: response.sell.transaction_id,
             };
-            this.root_store.notifications.addNotificationMessage(
-                contractSold(this.root_store.client.currency, response.sell.sold_for, Money)
-            );
+            if (window.location.pathname !== routes.trade || !this.root_store.ui.is_mobile) {
+                this.root_store.notifications.addNotificationMessage(
+                    contractSold(this.root_store.client.currency, response.sell.sold_for, Money)
```

Comment:
```suggestion
                    contractSold(this.root_store.client.currency, response?.sell?.sold_for, Money)
```

Reply:
hi @ali-hosseini-deriv this code was there before, i didn't change it and it was working fine without question marks :) why do we need them here? `response.sell` will be true when we reach this line